code/modules/power/lighting.dm

```diff
@@ -146,6 +146,37 @@
 	var/switchcount = 0
 	/// true if rigged to explode
 	var/rigged = FALSE
+	///looping sound for red alert alarm sound
+	var/datum/looping_sound/alarm_loop/lightalarm
```

Comment:
This isnt very performant and is probably going to overlap and maybe be earrapey